pkg/utils/logger/logger.go

```diff
@@ -13,69 +13,126 @@
 
 package logger
 
-import (
-	"fmt"
-	"os"
-	"strings"
+import "sync"
 
-	log "github.com/cihub/seelog"
+//Log is global variable so that log functions can be directly accessed
+var (
+	log  Logger
+	once sync.Once
 )
 
-const (
-	envLogLevel    = "AWS_VPC_K8S_CNI_LOGLEVEL"
```

Comment:
we need to keep the existing behaviour or allowing for log level and log filepath configuration via environment variables.